docs/book/src/cronjob-tutorial/testdata/finalizer_example.go

```diff
@@ -89,8 +89,8 @@ func (r *CronJobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 			// we can remove the finalizer and delete it.
 			// Note that you do not need to use the finalizer
 			// to remove any resource which is owned by the
-			// kind. Remember that we use by controllerutil.<method to set the ownership>
-			// More info: <add here the link for the k8s docs that explain the cascate delation>
+			// kind. Remember that we use by controllerutil.RemoveFinalizer
```

Comment:
controllerutil.RemoveFinalizer -> it is not the method that we set the ownership

Reply:
ok

Reply:
i commit this please  review